x-pack/legacy/plugins/canvas/public/application.tsx

```diff
@@ -94,13 +109,30 @@ export const initializeCanvas = async (
     },
   });
 
+  // TODO: We need this to disable the filtering modal from popping up in lens embeds until
+  // they honor the disableTriggers parameter
+  const action = startPlugins.uiActions.getAction(ACTION_VALUE_CLICK);
+
+  if (action) {
+    restoreAction = action;
+
+    startPlugins.uiActions.detachAction(VALUE_CLICK_TRIGGER, action.id);
+    startPlugins.uiActions.attachAction(VALUE_CLICK_TRIGGER, emptyAction);
```

Comment:
Do you get an error if you just remove the action?  If so we should probably fix that. It should be okay if a trigger is emitted with no actions attached. 

btw, I think you need to do this with `SELECT_RANGE_TRIGGER` too and `ACTION_SELECT_RANGE`.

Reply:
Yes, if there is no matching action, then it throws an error in the console, but it doesn't bubble up and throw a toast error in the ui that I saw.